app/models/member.rb

```diff
@@ -6,10 +6,12 @@ class Member < ActiveRecord::Base
   include DeviseTokenAuth::Concerns::User
   has_many :website_members, -> { uniq }, dependent: :destroy, autosave: true
   has_many :websites, through: :website_members
-  has_one :notification, dependent: :destroy, autosave: true
 
   validates :name, presence: true
   validates :email, presence: true, uniqueness: true
 
-  attr_accessor :confirm_success_url
+  def is_owner_of?(website)
+    WebsiteMember.with_role(:owner).where(website: website).map(&:member_id).include?(self.id)
```

Comment:
metoda asta verifica daca un membru este owner in tabela website_members atunci cand se apeleaza destroy in website_members_controller.rb pentru a nu permite stergerea lui